astropy/io/votable/converters.py

```diff
@@ -459,8 +490,14 @@ def _binparse_fixed(self, read):
     def _binoutput_var(self, value, mask):
         if mask or value is None or value == "":
             return _zero_int
+
+        if self.arraysize != "*" and len(value) > self.arraysize:
+            vo_warn(W46, ("unicodeChar", self.arraysize), None, None)
+            value = value[: self.arraysize]
```

Comment:
Ahh, review mistake, I didn't spot that this line still truncate and  is now inconsistent with the same method in `Char`

Reply:
Is a follow-up PR planned? Thanks!

Reply:
Oh I didn't notice that inconsistency, my bad!